base/setup/usetup/lang/es-ES.h

```diff
@@ -26,19 +24,19 @@ static MUI_ENTRY esESLanguagePageEntries[] =
     {
         4,
         3,
-        " Instalación de ReactOS " KERNEL_VERSION_STR " ",
+        " Instalaci�n de ReactOS " KERNEL_VERSION_STR " ",
```

Comment:
The revert actually broke the Spanish characters.

Reply:
The same mistake I made at first, then I realized that Usetup uses CP437 as encoding, and some chars (specially spanish accents) aren't in the same char codes than in windows-1252 or iso-8859-1